java/ci/Dockerfile.rocky

```diff
@@ -33,7 +33,7 @@ RUN dnf --enablerepo=powertools install -y  scl-utils gcc-toolset-${TOOLSET_VERS
 RUN mkdir /usr/local/rapids /rapids && chmod 777 /usr/local/rapids /rapids
 
 # 3.22.3+: CUDA architecture 'native' support + flexible CMAKE_<LANG>_*_LAUNCHER for ccache
-ARG CMAKE_VERSION=3.28.6
+ARG CMAKE_VERSION=3.30.7
```

Comment:
Typo?
```suggestion
ARG CMAKE_VERSION=3.30.4
```

Reply:
I was presuming we would want the latest version of 3.30 in these containers, and not the minimum required

Reply:
Will we change this again to cmake version 4 very soon?

Reply:
Okay. I didn't see 3.30.7 in https://pypi.org/project/cmake/#history but I see it in https://cmake.org/cmake/help/latest/release/3.30.html#id6. This is fine.

We probably won't update to cmake 4 for a while.